doc/manuals/code_contributions.md

```diff
@@ -51,6 +51,59 @@ Thanks to Alejandro Villamarin (published around October 2013):
           System.err.println("Failed. Reason is " + e.getMessage());
        }
 
+## JavaScript and NodeJS
+SmartSDK team developed and [SDK Client](https://github.com/smartsdk/ngsi-sdk-javascript) compatible with NSGIv2. 
+It allows to develop both NodeJS applicaitions and JavaScript applications.
+
+NodeJS:
+```javascript
+var NgsiV2 = require('ngsi_v2');
+
+var defaultClient = NgsiV2.ApiClient.instance;
+
+// Configure API key authorization: fiware_token
+var fiware_token = defaultClient.authentications['fiware_token'];
+fiware_token.apiKey = "YOUR API KEY"
```

Comment:
It is a good practise in JavaScript to include `;` at the end of the statements. In fact, you are following that criteria in most of your statement, but the following ones doesn't have it:

```
fiware_token.apiKey = "YOUR API KEY"
//fiware_token.apiKeyPrefix['X-Auth-Token'] = "Token"
var api = new NgsiV2.APIEntryPointApi()
```

The same in the client JavaScript fragment below.